app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+from shutil import copyfileobj
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+    source_filename, destination_filename = parts[1], parts[2]
```

Comment:
You do not need to perform unpacking with the use of indexes, write all results to a separate variable at once. Besides that, you will use the first element as well later in the code.
```suggestion
    command, source_filename, destination_filename = parts
```